clients/src/main/java/org/apache/kafka/clients/InFlightRequests.java

```diff
@@ -148,4 +151,10 @@ public int inFlightRequestCount() {
 
         return nodeIds;
     }
+    
+    public interface InFlightRequest {
```

Comment:
Ack. It seemed a little weird to tie it tightly to NetworkClient, but that is really the only place where it is used. It would be easy enough to add an interface in the future if we had another use.